backend/backend.go

```diff
@@ -640,39 +663,9 @@ func (backend *Backend) initPersistedAccounts() {
 				account.CoinCode, account.Code)
 			continue
 		}
+		backend.createAndAddAccount(
+			coin, account.Code, account.Name, account.Configurations, account.ActiveTokens)
 
-		// We split accounts if the user setting dictates it or if the keystore connected to the
-		// account does not support unified accounts.
-
-		var isBTCBased bool
-		switch account.CoinCode {
-		case coinpkg.CodeBTC, coinpkg.CodeTBTC, coinpkg.CodeRBTC, coinpkg.CodeLTC, coinpkg.CodeTLTC:
-			isBTCBased = true
-		}
-		splitAccounts := isBTCBased && (backend.config.AppConfig().Backend.SplitAccounts ||
```

Comment:
Also remove `SplitAccounts` field from the `Backend` struct in config/config.go?

Reply:
:heavy_check_mark: